receipt_upload/test/test_merchant_embedding_processor.py

```
@@ -0,0 +1,523 @@
+                    chromadb_bucket="test-bucket",
+                )
+
+                assert processor.places_client is None
+
+    def test_init_sets_queue_urls_in_environment(self):
+        """Test that queue URLs are set in environment."""
+        import os
+
+        with patch("receipt_upload.merchant_resolution.embedding_processor.DynamoClient"):
+            with patch("receipt_upload.merchant_resolution.embedding_processor.boto3"):
+                processor = MerchantResolvingEmbeddingProcessor(
```

Ruff (0.14.8)

Local variable `processor` is assigned to but never used

Remove assignment to unused variable `processor`

(F841)